package.json

```diff
@@ -28,6 +28,7 @@
     "stress:destroy": "docker-compose -f scripts/stress-test/docker-compose.yml stop"
   },
   "dependencies": {
+    "babel-plugin-transform-dynamic-import": "^2.1.0",
```

Comment:
@briandamaged How do you feel about introducing babel plugin for this?

Reply:
Hmm... I'll need to read through this PR more thoroughly to make sure I understand _how_ this plugin is being used.

(In particular: if the plugin is being used as a compilation step for `knex` itself, then that's probably fine.   However, if `knex` is becoming a top-level driver for `babel`, then it seems like that could become problematic.)